libs/sdk-ui-kit/src/Header/WorkspacePickerHomeFooter.tsx

```diff
@@ -0,0 +1,9 @@
+// (C) 2020 GoodData Corporation
+import React from "react";
+
+/**
+ * @internal
+ */
+export const WorkspacePickerHomeFooter: React.FC = ({children}) => {
+    return <div className={'home-footer-icon'}>{children}</div>;
```

Comment:
There is no need for curly brackets in className here. Also beware of the single quotes and add a newline to this file please.